packages/pool/contracts/StateUtils.sol

```diff
@@ -188,6 +200,19 @@ contract StateUtils is IStateUtils {
         emit SetDaoAgent(daoAgent);
     }
 
+    /// @notice Called after deployment to set the addresses of the DAO Voting
+    /// apps
+    /// @param _votingApps Addresses of the DAO Api3Voting apps
+    function setVotingApps(address[] calldata _votingApps)
```

Comment:
It's probably a safe assumption that we won't change the voting app addresses again. Let me know if you think this is a bad idea.

Reply:
I'm fine with that, however if we replace  `require(votingApps.length == 0, ERROR_UNAUTHORIZED)` with `onlyDaoAgent`  or a new similar modifier this might make it easier to migrate to a different DAO framework later on..
We might add a voting app with a special high requirements for the protocol upgrade in this issue https://github.com/api3dao/api3-dao/issues/141

Reply:
A potential upgrade in the future would involve leaving the current Aragon framework completely, so I don't see much advantage in being able to switch around components (given that we have a solid thing to work with, which I think we do). So I think that's more of a nice to have, but feel free to add it.

Reply:
> A potential upgrade in the future would involve leaving the current Aragon framework completely, so I don't see much advantage in being able to switch around components (given that we have a solid thing to work with, which I think we do). So I think that's more of a nice to have, but feel free to add it.

exactly, what would require just to change the `votingApps` addresses, instead of redeploying the entire `Pool` contract

Reply:
Okay, I think we can do this in the PR related to https://github.com/api3dao/api3-dao/issues/147